index.html

```diff
@@ -5,8 +5,119 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>concert</title>
+    <!-- stylesheet -->
+    <link rel="stylesheet" href="./assets/css/styles.css">
+    <!-- font awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 <body>
-    <h1>concert</h1>
+    <header id="navigation">
+        <h4 class="logo  fa-shake"><span class="half">Con</span>cert</h4>
+        <nav>
+            <div class="menu-toggle"></div>
+            <div class="mobile-menu">
+                <ul class="nav-list">
+                    <li class="nav-content">
+                        <a href="#">home</a>
+                      </li>
+                    <li class="nav-content">
+                        <a href="./about.html">about us</a>
+                      </li>
+                    <li class="nav-content">
+                        <a href="#main-program">program</a>
+                      </li>
+                    <li class="nav-content">
+                        <a href="#performing-artist">artist</a>
+                      </li>
+                    <li class="nav-content">
+                        <a href="#" class="tickets">buy Tickets</a>
+                      </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
```

Comment:
- [ ] Nice work implementing the "Navbar" It looks really good :clap: However you did not implement the top dark bar that contains the social links. Kindly make sure all the sections in the navbar are implemented as in the provided design, which will show your ability to understand and create a UI base on a medium-fidelity wireframe.
### Your Navbar design :point_down: 
![Screenshot from 2023-05-22 23-35-04](https://github.com/Othniel2471/capstone-html-css/assets/91142432/02b1ca44-554c-4bc4-ad74-1924afe6e069)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-05-22 23-47-20](https://github.com/Othniel2471/capstone-html-css/assets/91142432/c579bff2-279c-4b0a-b7e8-11488f663f85)